tests/ticket2818.py

```diff
@@ -23,19 +23,19 @@
 from __future__ import print_function
 import os
 import sys
-sys.path = [os.path.join(os.path.abspath(os.path.dirname(__file__)), "ticket2818")] + sys.path
+sys.path = [os.path.join(os.path.abspath(os.path.dirname(__file__)), "ticket2818helper")] + sys.path
```

Comment:
I understand the use of `ticket2818helper` in a previous commit now, but it should be part of this commit.